README.md

```diff
@@ -4,7 +4,7 @@
 </div>
 
 # pysat: Python Satellite Data Analysis Toolkit
-[![Build Status](https://travis-ci.com/pysat/pysat.svg?branch=master)](https://travis-ci.com/pysat/pysat)
+[![Build Status](https://travis-ci.org/pysat/pysat.svg?branch=main)](https://travis-ci.org/pysat/pysat)
```

Comment:
```suggestion
[![Build Status](https://travis-ci.com/pysat/pysat.svg?branch=main)](https://travis-ci.com/pysat/pysat)
```

Reply:
I'm not sure why this is still showing a difference between main and master here. I think the diff is a bit out of date. Perhaps the tests on the new commits in the other branch need to run first?